src/functionalTest/java/io/knotx/stack/functional/FailedKnotxFragmentScenarioTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.stack.functional;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import io.knotx.junit5.KnotxApplyConfiguration;
+import io.knotx.junit5.KnotxExtension;
+import io.knotx.junit5.RandomPort;
+import io.knotx.junit5.wiremock.ClasspathResourcesMockServer;
+import io.knotx.stack.KnotxServerTester;
+import io.vertx.junit5.VertxTestContext;
+import io.vertx.reactivex.core.Vertx;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(KnotxExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class FailedKnotxFragmentScenarioTest {

```

Comment:
What do you think about moving those scenarios to unit tests in knotx-fragments. Here, in knotx-stack, we can add new or update the `KnotxFragmentsDebugDataWithHandlebarsTest` scenario to have a failed fragment. Please note that we can do this with additional file in `KnotxApplyConfiguration`.

`KnotxFragmentsDebugDataWithHandlebarsTest` -> `KnotxFragmentsExecutionLogsTest`

Reply:
If I understand correctly then I don't agree. This functionality is independent from `debug` switch so I don't think they need to be in the same test suite

Reply:
You are right :)